source/NVDAObjects/__init__.py

```diff
@@ -421,9 +421,8 @@ def _get_name(self):
 	#: Type definition for auto prop '_get_role'
 	role: int
 
-	def _get_role(self) -> int:
+	def _get_role(self) -> controlTypes.Role:
 		"""The role or type of control this object represents (example: button, list, dialog).
-		@return: a ROLE_* constant from L{controlTypes}

```

Comment:
This comment should be updated as well to explain that now the role value comes from the enum

Reply:
I don't know what value that provides, the return type should make that clear hopefully